app/assets/stylesheets/style.scss

```diff
@@ -7,6 +7,22 @@
   font-size: rem-calc(14);
 }
 
-.advanced-search li {
-  list-style: none;
+.has-form {
+ padding: 0px 0px 0px 0px;
+}
+
+.submit {
+ padding: 0px 0px 0px 0px;
+}
+
+.profile-info {
+  .profile_img {
+  border-radius: 50%;
+  width: 40px;
+  }
+
+  .profile_name {
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.